apps/api-documenter/src/documenters/YamlDocumenter.ts

```diff
@@ -96,12 +90,13 @@ export class YamlDocumenter {
     return {
       name: 'SharePoint Framework reference',
       href: '~/overview/sharepoint.md',
-      items: [ ]
+      items: []
     };
   }
 
   /** @virtual */
-  protected onCustomizeYamlItem(yamlItem: IYamlItem): void { // virtual
+  protected onCustomizeYamlItem(yamlItem: IYamlItem): void {
```

Comment:
Should the `// virtual` comment even be here?

Reply:
Nope.  `// virtual` was my old convention before TSDoc had `@virtual`. Looks like I forgot to delete it in [`eaf8b5`](https://github.com/microsoft/web-build-tools/commit/eaf8b55659c1112307e3773bb9a1a39d4500ba25#diff-63cb599119dc3cf0dfa99bfa3ecc8f15)